types/params.go

```diff
@@ -57,7 +57,7 @@ type EvidenceParams struct {
 // ValidatorParams restrict the public key types validators can use.
 // NOTE: uses ABCI pubkey naming, not Amino names.
 type ValidatorParams struct {
-	PubKeyTypes []string `json:"pub_key_types"`
+	PubKeyType string `json:"pub_key_type"`
```

Comment:
JSON schema breakage as well. We might need to take this into account in RPC versioning, and we still need the old JSON schema for pre-1.0 nodes.

Reply:
will revert and treat length as always 1